src/popup.html

```diff
@@ -1,9 +1,52 @@
 <html>
   <head>
     <title>NBGitPuller</title>
```

Comment:
style component contains all the CSS and classes for all the elements added through index.jsx